styles/pages/contact.scss

```diff
@@ -0,0 +1,27 @@
+.contact {
+  &__faq{
+    margin-top: 10px;
+  }
+
+  &__methods {
+    text-align: center;
+    margin: 50px 0 30px;
+
+    &__link {
+      display: inline-block;
+      margin: 20px;
+      max-width: 250px;
+      background-color: $accessible-red;
+      padding: 20px;
+      color: $white;
+      font-weight: bold;
+      text-decoration: none;
+
+      svg {
```

Comment:
Might be worth using a class so that it still works if you change to using an `<img>` instead of an `svg`.